src/internal/modules/billing/routes/bill-run.js

```diff
@@ -211,7 +211,9 @@ const routes = {
       validate: {
         params: Joi.object().keys({
           batchId: Joi.string().uuid(),
-          invoiceId: Joi.string().uuid()
+          invoiceId: Joi.string().uuid(),
+          originalInvoiceId: Joi.string().uuid(),
+          rebillInvoiceId: Joi.string().uuid()
```

Comment:
`originalInvoiceId: Joi.string().uuid().optional(),
   rebillInvoiceId: Joi.string().uuid().optional()`
should be received as query parameters instead so that you can use the same route to delete normal as well as rebilling invoices.